ops/ccs-ops-misc/load_test/lambda/server-regression/.dockerignore

```diff
@@ -0,0 +1,7 @@
+*
+!common/
+!config/
+!v1
+!v2
```

Comment:
`common` and `config` have a trailing slash, but `v1` and `v2` don't. Does this cause different behavior in the `.dockerignore`?

Reply:
[There definitely is](https://stackoverflow.com/questions/17888695/difference-between-gitignore-rules-with-and-without-trailing-slash-like-dir-an):

>Yes, these rules are different.
>
>    * `/dir` will match a file, directory, link, anything named `dir`
>    * `/dir/` will match only a directory named `dir`
>    * `/dir/*` will match all files, directories and anything else inside a directory named `dir` (but not the `dir` directory itself).

This is definitely _confusing_ in the sense that these lines are not following a single convention for un-ignoring a directory. _However_, for now this will not cause any problems, and I would say that both styles are essentially interchangeable as far as what we are using the `.dockerignore` for.